constrained-voucher.mkd

```diff
@@ -195,6 +197,15 @@ The MASA therefore knows if the pledge supports PKIX operations, PKIX format cer
 Based upon this, the MASA can select which attributes to use in the voucher for certain operations, like the pinning of the Registrar identity.
 This is described in more detail in {{yang-voucher}}, {{pinning}} and {{pinned-with-rpk}} (for RPK specifically).
 
+# Updates to RFC8366 and RFC8995
+
+This section details the ways in which this document Updates.
+The terminology for Updates is taken from {{I-D.kuehlewind-update-tag}}.
+
+This document Updates {{RFC8366}}. It Extends {{RFC8366}} by creating a new serialization format.
+
+This document Updates {{RFC8995}}. It Amends {{RFC8995}} by clarifying how pinning is done, and ???.
```

Comment:
suggestion to change the '???' here to '???TBD' so that we can easily find pending work items by searching for the string 'TBD'.